### PR TITLE
Fixed a few spelling mistakes in Luxembourgish translation

### PR DESCRIPTION
**Pull request by mweimerskirch**
Fixed a few spelling mistakes in Luxembourgish translation and also adjusted trans-unit "51" to commit #21335

| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? |no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 1]
```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Et gouf keen temporären Dossier an der php.ini konfiguréiert.</target>
+                <target>Et gouf keen temporären Dossier an der php.ini konfiguréiert oder den temporären Dossier existéiert net.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: Are you sure we should merge this? This is not a pure translation, but adds a hint that might not be correct as it doesn't exist in the source message.

**Review by nicolas-grekas — COMMENTED** [review]
Should be merged on branch 2.7

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Validator/Resources/translations/validators.en.xlf#L195 :+1:

**Comment by nicolas-grekas** [comment]
Thank you @mweimerskirch.

**Pull request closed without merging**